# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/green.png
# Commit: b72d1d6131cc
# Full Hash: b72d1d6131cce6845af6ffd6f83de7f993930961
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-26 09:42:55
# Description:
#   Bug 1576154 - Check mSVGDocumentWrapper when trying to report use counters; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43256
# ==============================================================================

diff -r 929421783c5b -r b72d1d6131cc dom/base/test/green.png
Binary file dom/base/test/green.png has changed